### PR TITLE
Updated locale file, since 'message' is missing in dialog

### PR DESCRIPTION
**Pull request by Znow**

### CODE REVIEW

File: config/locales/da.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ da:
         admin:
           users:
             delete: Slet bruger
+              message: Er du sikker på du vil slette brugeren?
```
Comment (parndt) [inline config/locales/da.yml]: this indentation seems out, or perhaps you meant to nest it under the other delete? https://github.com/refinery/refinerycms-authentication-devise/blob/master/config/locales/en.yml#L57

Comment (Znow) [inline config/locales/da.yml]: Yes I did.

Comment (parndt) [inline config/locales/da.yml]: yes, it would be nice if time allowed instant feedback 😉

**Comment by bricesanchez** [comment]
Closed in favor of #35

**Pull request closed without merging**